content/07.pathogenesis.md

```diff
@@ -10,6 +10,36 @@ This section would also be great for the introduction of zoonotic diseases which
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
 
+Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052]. 
+
+The S protein is a highly glycosylated trimer that requires two proteolytic cleavage events, leading to substantial conformational changes, to achieve viral fusion with the host cell membrane [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507]. 
+
+Each protomer is composed of an S1 and an S2 subunit, which mediate receptor binding and viral fusion, respectively. 
+
+The priming proteolytic events occur sequentially, first at the S1/S2 junction and then at the S2’ site, ultimately resulting in the shedding of the S1 subunit and transitioning of the S2 subunit to a more stable, fusion-conducive conformation. 
+
+Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases it can use to cleave the S protein [@doi:10.1016/j.cell.2020.02.052]. 
+
+Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry in vitro [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9]. 
+
+Interestingly, SARS-CoV-2 S protein also contains a RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV [@doi:10.1038/s41586-020-2012-7]. 
+
+Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10.1016/S0092-8674(00)81771-7; @doi:10.1006/viro.1999.9716]. 
```

Comment:
```suggestion
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716]. 
```
DOIs with parentheses cause a bug -- if you want, you can convert these on the website shortdoi.org. But I didn't warn you about that so not your fault!